sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitTestFileReport.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.php.phpunit;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.sonar.api.batch.fs.FilePredicates;
+import org.sonar.api.batch.fs.FileSystem;
+import org.sonar.api.batch.fs.InputFile;
+import org.sonar.api.batch.sensor.SensorContext;
+import org.sonar.api.measures.CoreMetrics;
+import org.sonar.api.utils.ParsingUtils;
+import org.sonar.plugins.php.api.Php;
+import org.sonar.plugins.php.phpunit.xml.TestCase;
+
+/**
+ * The PhpUnitTestFileReport contains all the results of test cases appearing in a given test file.
+ * The reason why the report is file-based (as opposed to class-based) is that the SonarQube measures
+ * are stored per file.
+ */
+public class PhpUnitTestFileReport {
+
+  public PhpUnitTestFileReport(String file, double testDuration) {
+    this.file = file;
+    this.testDuration = testDuration;
+  }
+  private static final Logger LOGGER = LoggerFactory.getLogger(PhpUnitTestResultImporter.class);
+  private int errors = 0;
+  private int failures = 0;
+  private String file;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1213)